cmd/examples/test-dataplane/test-dataplane.go

```diff
@@ -21,13 +21,19 @@ import (
 	"log"
 	"net"
 	"os"
+	"path"
```

Comment:
A more general question: Is the relationship of NSMD <-> localMechanism 1-1 or can an NSMD support multiple localMechanism. If it can support multiple localMechanisms, do we include the ability for an NSMD instance to register multiple localMechanisms?